inc/saiswitch.h

```diff
@@ -3272,6 +3312,8 @@ typedef struct _sai_switch_api_t
     sai_remove_switch_tunnel_fn            remove_switch_tunnel;
     sai_set_switch_tunnel_attribute_fn     set_switch_tunnel_attribute;
     sai_get_switch_tunnel_attribute_fn     get_switch_tunnel_attribute;
+    sai_switch_mdio_cl22_read_fn           switch_mdio_cl22_read;
+    sai_switch_mdio_cl22_write_fn          switch_mdio_cl22_write;
```

Comment:
It can be:
```
sai_switch_mdio_read_fn          switch_mdio_cl22_read
sai_switch_mdio_write_fn         switch_mdio_cl22_write
```

Reply:
In the first commit: https://github.com/opencomputeproject/SAI/pull/1507/commits/912c4c4e5c73808e06f535677e82e99b46500117
The new cl22 functions had the type you suggested. But it would not pass the parse.pl sanity check. I changed the cl22 function to the new type in the last commit: https://github.com/opencomputeproject/SAI/pull/1507/commits/0fb1c06d137eff057f61c8ef9fef0ea234514da5